pxtextension.ts

```diff
@@ -54,14 +63,21 @@ class MlEvent {
   }
 
   //% blockId=mlrunner_is_ml_event
-  //% block="is %this action"
+  //% block="is $this action"
   isEvent(): boolean {
     if (!mlrunner.isRunning()) {
       mlrunner.startRunning();
       return false;
     }
     return this.eventValue == mlrunner.currentActionId();
   }
+
+  //% blockId=mlrunner_is_on_event_stop
+  //% block="on $this stop $duration"
+  //% draggableParameters="reporter"
```

Comment:
`//% block="on $this stop $duration"` - the `$this` combined with `$duration` and `//% draggableParameters="reporter"` don't work together. I'm not sure if this is a bug, but I will do some further exploration.

Reply:
It now seems fine with [092b9a2](https://github.com/microbit-foundation/pxt-ml-extension-poc/pull/10/commits/092b9a26bf7b473e8ac663babc72c5f1cda05ce1). Will do more testing on Monday.